app/sass/pages/_api.scss

```diff
@@ -102,27 +107,14 @@
 }
 
 
-
-/**
- * API | Main Content
- *
- * @class     .api__content
- * @class     .api__content__body
- * @class     .api__content__body__arguments
- *
- * @class     .api__content__body__argument
- * @selector  {::after}
- * @selector  {:last-of-type}
- * @selector  {:nth-child}
- *
- * @class     .api__content__example
- */
-
 .api__content {
-  display: grid;
   float: right;
-  grid-template-columns: repeat(auto-fit, minmax(50%, 1fr));
   width: calc(100% - 200px);
+}
+
+.api__documentation {
```

Comment:
There needs to be a top border here, preferably using the `:before` pseudo-selector so that it sits flush with the background of the first `.api__content__example`.

Reply:
good call, I think `/api` should just send people to `/resources`. I've updated it as such.